core/src/memory/default_resource_polyfill.cpp

```diff
@@ -0,0 +1,57 @@
+#include <atomic>
```

Comment:
You didn't put a copyright statement into the file...

Reply:
Ah yeah, sure.